lib/phraseapp_updater/locale_file/json_file.rb

```diff
@@ -0,0 +1,29 @@
+require 'multi_json'
+require 'oj'
+
+# We're working with pure JSON, not
+# serialized Ruby objects
+Oj.default_options = {mode: :strict}
+
+class PhraseAppUpdater
+  class LocaleFile
+    class JSONFile < LocaleFile
+      EXTENSION = "json"
+      def self.from_hash(name, hash)
+        new(name, MultiJson.dump(hash))
+      end
+
+      def parse(content)
+        MultiJson.load(content)
+      rescue MultiJson::ParseError => e
+        raise ArgumentError.new("Provided content was not valid JSON")
```

Comment:
That's a good point. I'll look into it, but printing the cause either way sounds like a good plan.